src/js/modules/plant/overlay.js

```diff
@@ -69,17 +69,14 @@ export default View.extend({
       const $img = $container.children('img');
       const height = $img.height();
       const width = $img.width();
-      const pos = $container.position();
-      let top = pos.top;
-      const left = pos.left * scale;
+      const pos = object.moveable.getPosition();
```

Comment:
> separating actually planted objects from objects available in tollbox (or palette, that's the name introduced in streetmix)

That's a must!

I've been thinking about something like this:

```javascript
function moveable({ view }) {}
function draggable({ view, handler }) {
  onDrag(event) {
    if (handler) {
      handler.moveTo({ x, y });
      event.preventDefault();
    }
  }
}

View.extend({
  initialize() {
    this.moveable = moveable({ view: this });
    
    if (draggable) {
       draggable({ view: this, handler: this.moveable });
    }
  }
})
```

What do you think?